molecule/default/molecule.yml

```diff
@@ -0,0 +1,26 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+  # name: podman
+platforms:
+  - name: instance
+    image: "geerlingguy/docker-${MOLECULE_DISTRO:-fedora35}-ansible:latest"
+    # command: /sbin/init
+    command: ${MOLECULE_DOCKER_COMMAND:-""}
+    # tmpfs:
+    #   - /run
+    #   - /tmp
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
+provisioner:
+  name: ansible
+verifier:
+  name: ansible
+# lint: |
```

Comment:
we could run the linters here; this means that running "molecule test" will automatically run linters.  Otherwise we have to run them separately.

Reply:
As long as the top-level linter will check all of the files across all of the target platforms, this is fine to leave commented out so we only run the lint check once.